incubator/hnc/e2e/tests/delete-anchor-crd_test.go

```diff
@@ -0,0 +1,35 @@
+package test
+
+import (
+	"os/exec"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Delete-anchor-crd", func() {
+	It("should create parent and deletable child", func() {
+		// create a namespace delete-crd-parent
+		cmd := exec.Command("kubectl", "create", "ns", "delete-crd-parent")
+		Expect(cmd.Run()).Should(BeNil())
+
+		// create a namespace delete-crd-child and set its parent as delete-crdparent
+		cmd = exec.Command("kubectl", "hns", "create", "delete-crd-child", "-n", "delete-crd-parent")
+		Expect(cmd.Run()).Should(BeNil())
+
+		// allow cascading delete for delete-crd-child
+		cmd = exec.Command("kubectl", "hns", "set", "delete-crd-child", "--allowCascadingDelete")
+		Expect(cmd.Run()).Should(BeNil())
+	})
+
+	It("should delete the CRD", func() {
```

Comment:
Each "It()" block should be a complete test, and also there's no guarantee that they're run in order AFAIK. So this should all be one single "It()" block.

Reply:
Done